modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/model/hdbtable/XSKDataStructureHDBTableCommonColumnModel.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.model.hdbtable;
+
+public class XSKDataStructureHDBTableCommonColumnModel {
+
+  private String length;
+
+  private String precision;
+
+  private String scale;
+
+  private String sqlType;
+
+  private String defaultValue;
+
+  private boolean isDefaultValueDateTimeFunction;
+
+  public XSKDataStructureHDBTableCommonColumnModel() {
+  }
+
+  public XSKDataStructureHDBTableCommonColumnModel(String length, String precision, String scale, String sqlType, String defaultValue,
```

Comment:
I think that the same reasoning goes for most XSKDataStructureModels but if we only add it here, we need to refactor the parsing logic since there are multiple places, in which the setters are used. I don't think that this will bring any benefits in this case since:
1. Incorrect object state - they are only meant to be used in one place. This constructor here is not being called outside of its children's super() calls and even then, these constructors are used only for unit tests and the setters are used when parsing the HDBDD.
2. Immutability - not needed here
3. Readability - It will improve the readability of the tests, since they are using the constructors but not inside the main logic.